package.json

```diff
@@ -16,22 +16,29 @@
     "precompile-svgs": "node utils/precompileSvgs/index.js && yarn run pregenerate",
     "precompile-custom-svgs": "node utils/precompileSvgs/index.js --custom && yarn run pregenerate",
     "_lint-watch-fix": "yarn lint -w -m",
-    "test": "jest --config=jest.conf/index.js",
+    "test": "yarn test:unit && yarn test:visual",
+    "test:unit": "jest --config=jest.conf/index.js",
+    "test:visual": "JEST_PUPPETEER_CONFIG=./jest-puppeteer.config.js node startVisualTests.js",
     "_api-watch": "chokidar \"**/lib/**\" -c \"node utils/extractApi.js\""
   },
   "files": [
     "lib"
   ],
   "dependencies": {
+    "@percy/cli": "^1.28.7",
+    "@percy/puppeteer": "^2.0.2",
     "@vue/composition-api": "1.7.2",
     "aphrodite": "https://github.com/learningequality/aphrodite/",
     "autosize": "3.0.21",
     "css-element-queries": "1.2.0",
     "date-fns": "1.30.1",
     "frame-throttle": "3.0.0",
     "fuzzysearch": "1.0.3",
+    "jest-puppeteer": "^10.0.1",
     "lodash": "4.17.21",
     "popper.js": "1.16.1",
+    "ps-tree": "^1.2.0",
+    "puppeteer": "^22.11.0",
```

Comment:
These new dependencies should be on the `devDepencencies` prop, since they are just for development purposes.

Reply:
Sure. Will change that accordingly :+1: